tests/test_all.py

```diff
@@ -291,7 +292,19 @@ def test_zypper_verify_passes(container_per_test: ContainerData) -> None:
     [
         c
         for c in ALL_CONTAINERS
-        if (c not in [INIT_CONTAINER, PCP_CONTAINER] + POSTGRESQL_CONTAINERS)
+        if (
+            c
+            not in (
+                [
+                    INIT_CONTAINER,
+                    PCP_CONTAINER,
+                    # kernel-module-container contains systemd due to pesign,
+                    # fixes are pending
+                    KERNEL_MODULE_CONTAINER,
+                ]
+                + POSTGRESQL_CONTAINERS
+            )
+        )
```

Comment:
I would use a set here like this:

```suggestion
        if c not in {INIT_CONTAINERS, PCP_CONTAINER, KERNEL_MODULE_CONTAINER} | {POSTGRESQL_CONTAINERS}
            c
            not in (
                [
                    INIT_CONTAINER,
                    PCP_CONTAINER,
                    # kernel-module-container contains systemd due to pesign,
                    # fixes are pending
                    KERNEL_MODULE_CONTAINER,
                ]
                + POSTGRESQL_CONTAINERS
            )
        )
```

Reply:
@ricardobranco777 this isn't working (and there  is a typo INIT_CONTAINERS->INIT_CONTAINER) because we're mixing sets and lists here (e.g. postgresql_containers is a list, so {[1,2,3]} gives

```
>>> {[1,2,3]}
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unhashable type: 'list'
```

Reply:
> @ricardobranco777 this isn't working (and there is a typo INIT_CONTAINERS->INIT_CONTAINER) because we're mixing sets and lists here (e.g. postgresql_containers is a list, so {[1,2,3]} gives
> 
> ```
> >>> {[1,2,3]}
> Traceback (most recent call last):
>   File "<stdin>", line 1, in <module>
> TypeError: unhashable type: 'list'
> ```

You can use `set(POSTGRESQL_CONTAINERS)`.

Reply:
@ricardobranco777 same issue:

```
tests/test_all.py:298: in <listcomp>
    {
<string>:3: in __hash__
    ???
E   TypeError: unhashable type: 'list'
```

I don't have time to debug this right now. lets take this into a followup PR.